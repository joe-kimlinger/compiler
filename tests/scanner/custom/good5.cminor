// Coments
,
/* This is a multi-
line comment
that should be picked up as one*/
;
/**/
beforeComment/*******comment*******/afterComment
