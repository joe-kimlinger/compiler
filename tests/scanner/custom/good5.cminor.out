COMMENT
COMMA
COMMENT
SEMICOLON
COMMENT
IDENTIFIER beforeComment
COMMENT
IDENTIFIER afterComment
